<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FpsChangeDialog</class>
 <widget class="QDialog" name="FpsChangeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>183</width>
    <height>70</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>183</width>
    <height>70</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>183</width>
    <height>70</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FPS módosítása</string>
  </property>
  <property name="windowIcon">
   <iconset resource="re.qrc">
    <normaloff>:/files/Resources/ikon.png</normaloff>:/files/Resources/ikon.png</iconset>
  </property>
  <widget class="QLineEdit" name="fpsValueLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="fpsLabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>14</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>fps</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Mégsem</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="re.qrc"/>
 </resources>
 <connections/>
</ui>
